docs/get-started/utxos/overview.md

```diff
@@ -0,0 +1,17 @@
+---
+id: overview
+sidebar_position: 1
+title: Overview
+sidebar_label: Overview
+description: A full-stack Web3 infrastructure platform purpose-built for UTXO blockchains like Bitcoin and Cardano
+image: /img/og/og-getstarted-utxos.png
+---
+
+Web3 adoption is accelerating with more users and businesses exploring the potential of blockchain technology. However, the onboarding experience to web3 is still daunting, and a critical reason is the complexity of blockchain. From setting up a wallet with seed phrases, to needing crypto to pay for transaction fees; this is a significant barrier to entry and the user experience could be better on multiple fronts.
+
+| List of Features |  |  |
+|----|---|
+| [Wallet As A Service](https://utxos.dev/wallet-as-a-service) | Social login wallets with seamless user experience | [Docs](https://docs.utxos.dev/wallet) |
+| [Transaction Sponsorship](https://utxos.dev/transaction-sponsorship) | Sponsor transaction fees for your users and improve onboarding experience | | [Docs](https://docs.utxos.dev/sponsor) |
```

Comment:
FYI this table doesn't render, I think, because the number of columns are different between the rows.